code/game/machinery/iv_drip.dm

```diff
@@ -217,7 +217,9 @@
 	if(!isliving(usr))
 		to_chat(usr, "<span class='warning'>You can't do that!</span>")
 		return
-
+	if(isanimal(usr))
+		var/mob/living/simple_animal/user_animal = usr
+		return user_animal.canUseTopic()
```

Comment:
This doesn't make sense, what are you returning this for?

Did you mean to let it through they can use topics?

Reply:
I was under the impression canUseTopic is the check for machine interaction, so I left it in the return in case there are some weird edge cases